cmd/dev.go

```diff
@@ -515,3 +519,36 @@ func init() {
 	devRunCmd.Flags().StringP("dir", "d", ".", "The directory to run the development server in")
 	devRunCmd.Flags().String("websocket-id", "", "The websocket room id to use for the development agent")
 }
+
+func displayLocalInstructions(port int, agents []project.AgentConfig) {
+	title := tui.Title("🚀 Local Agent Interaction")
+
+	// Combine all elements with appropriate spacing
+	fmt.Println()
+	fmt.Println(title)
+
+	// Create list of available agents
+	if len(agents) > 0 {
+		fmt.Println()
+		fmt.Println(tui.Bold("Available agents:"))
+
+		for _, agent := range agents {
+			// Display agent name and ID
+			fmt.Println(tui.Text("  • " + agent.Name))
+			fmt.Println(tui.Secondary("    ID: " + agent.ID))
```

Comment:
you could use tui.Paragraph and push to it. i also have been added the ID on same line as muted with the left side pad aligned. but this looks great and i can tweak it!